### ISSUE TITLE
npm install instructions broken

### ISSUE THREAD

**Issue by mgkimsal** [canjs/canjs#5161]
https://canjs.com/doc/guides/setup.html#Webpack

`npm install can@5 --save`

This will return

```
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for can-connect@3.2.1: wanted: {"npm":"^3.0.0"} (current: {"node":"8.12.0","npm":"6.10.1"})
npm ERR! notsup Not compatible with your version of node/npm: can-connect@3.2.1
```

The can-connect has a hard-coded dependency on npm@3.

`npm install can@5 --save --engine-strict=false`

gets past this, but... something needs to change.  I'm not sure if the docs need to be amended, or can-connect needs to be fixed.

**Comment by justinbmeyer** [comment]
thanks for reporting! We will fix this soon.

**Comment by mgkimsal** [comment]
I suspect the bigger fix might be changing ^3.0.0 in can-connect to >=3.0.0.

thanks @justinbmeyer and @cherifGsoul  for reacting so quickly!
